release/preview/alpine/docker/Dockerfile

```diff
@@ -0,0 +1,97 @@
+# Docker image file that describes an Alpine3.8 image with PowerShell installed from .tar.gz file(s)
+# NOTE: the alpine tar.gz when this was written doesn't contain the modules.  For that we need a container with modules.
+# To accomplish this, we will get the modules from the full linux tar.gz package, then
+# overlay the alpine tar.gz on top of it.
+# There are TODO's in the file on updates that should occure one the Alpine .tar.gz contains everything
+
+# Define arg(s) needed for the From statement
+ARG fromTag=3.8
+
+FROM alpine:${fromTag} AS installer-env
+
+# Define Args for the needed to add the package
+ARG PS_VERSION=6.1.0-rc.1
+# TODO: once the official build produces a full package for alpine, update this to the full package
```

Comment:
This comment is kind of misleading as the next line IS the full package. Or am i missing something?